x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -113,6 +114,18 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
       ];
     }, [os]);
 
+    const operatorOptions = useMemo<Array<EuiSuperSelectOption<string>>>(() => {
```

Comment:
Is there a specific reason to use `useMemo` here?

Reply:
Not really. I was experimenting 😄 This should be just an array of options.